system-tests/outputFile/outputFile.test.js

```diff
@@ -43,7 +43,7 @@ describe('outputFile', () => {
 		childProcess.on('close', function() {
 			return readFileAsync(directionPath, 'utf-8').then((content) => {
 				expect(content).toEqual(systemTestUtils.stripColors(stdout));
-				expect(content).toMatchSnapshot();
+				expect(content.replace('×', '✖')).toMatchSnapshot();
```

Comment:
@hudochenkov I am curious about this fix =) could you explain where charset is wrong?

Reply:
I have no idea what happened. But Jest was telling that two lines are different (Windows output and snapshot), and this symbol was the difference.

Reply:
Looks like we need to use [this util](https://github.com/stylelint/stylelint/blob/master/lib/formatters/__tests__/prepareFormatterOutput.js) here

Reply:
Too much trouble, in my opinion. We would need to move this utility somewhere, then change this test, and test that used this utility. I think it's ok to keep things as they are for less complex codebase.